sentry-delayed_job/spec/sentry/delayed_job_spec.rb

```diff
@@ -234,6 +237,93 @@ def perform
         expect(event[:contexts][:"Active-Job"][:job_class]).to eq("FailedJob")
       end
     end
+
+    context "when tracing is enabled" do
+      before do
+        perform_basic_setup do |config|
+          config.traces_sample_rate = 1.0
```

Comment:
you also need this line here:

```ruby
          config.rails.skippable_job_adapters << "ActiveJob::QueueAdapters::DelayedJobAdapter"
```

the 2 extra events are from the `ActiveJob` integration. 

in the earlier tests it's disabled with

https://github.com/getsentry/sentry-ruby/blob/53f0bd9a0beb5c742b7979135f70f5ff98d82021/sentry-delayed_job/spec/sentry/delayed_job_spec.rb#L182-L184

but because this block is evaluated after that, you need to register the adapter again.

Reply:
Aha! thanks, fixed.